tests/test_sample.py

```diff
@@ -213,13 +215,11 @@ def test_pauliz_hadamard(self, device, shots, tol):
         dev.apply("CNOT", wires=[1, 2], par=[])
 
         dev._obs_queue = [
-            qml.PauliZ(wires=[0], do_queue=False)
-            @ qml.Hadamard(wires=[1], do_queue=False)
-            @ qml.PauliY(wires=[2], do_queue=False)
+            Tensor(["PauliZ", "Hadamard", "PauliY"], [[0], [1], [2]], [[], [], []], qml.operation.Sample)
         ]
 
-        for idx in range(len(dev._obs_queue)):
-            dev._obs_queue[idx].return_type = qml.operation.Sample
+        # for idx in range(len(dev._obs_queue)):
+        #     dev._obs_queue[idx].return_type = qml.operation.Sample
```

Comment:
ditto

Reply:
Oh I think I left it commented just in case PL core requires this logic again :laughing: